packages/lpa-questionnaire-web-app/src/controllers/accuracy-submission.js

```diff
@@ -2,7 +2,7 @@ const logger = require('../lib/logger');
 const { VIEW } = require('../lib/views');
```

Comment:
i had to rip up the createOrUpdate concept; it was burying the logic where i didn't want it. once i'd cut it in half it was pretty search-and-replace stuff to mend a lot of the stuff below